<?xml version="1.0"?>
<launch>
    <node name="CAD_loader" pkg="elvez_pkg" type="load_files.py" output="screen">
    </node>
</launch>  
